packages/design-system/src/__theme__/default/index.css

```diff
@@ -129,13 +130,14 @@
   --ads-v2-colors-content-surface-warning-bg: var(--ads-v2-color-bg-warning);
   --ads-v2-colors-content-surface-special-bg: var(--ads-v2-color-bg-special);
   --ads-v2-colors-content-surface-premium-bg: var(--ads-v2-color-bg-premium);
-  --ads-v2-colors-content-surface-default-border: var(--ads-v2-color-border);
+  --ads-v2-colors-content-surface-default-border: var(--ads-v2-color-border-muted);
```

Comment:
This change impacts 

- popover border
- menu separator
- datepicker border and separator
1. 
![Screenshot 2023-08-18 at 5 22 06 PM](https://github.com/appsmithorg/design-system/assets/13763558/c4e93d61-ad23-4a5d-a77b-2e89972f7003)

Reply:
@albinAppsmith - why is datepicker styled using content tokens? Shouldn't it be using control tokens?

Reply:
Clarified with @albinAppsmith over call that only the popover part of date pick is using the content token, the rest is using control.